index.html

```diff
@@ -35,15 +35,19 @@
 a third party
 easier and more secure on the Web.</p>
 
-<p>We produce standards that are protected by the <a 
-href="https://www.w3.org/2004/01/pp-impl/83482/status">W3C Royalty-free Patent Policy</a>. This 
+<p>We produce standards that are protected by the <a
+href="https://www.w3.org/2004/01/pp-impl/83482/status">W3C Royalty-free Patent Policy</a>. This
 ensures that W3C standards will remain free for everyone to use, now and in the future.</p>
 
+<p data-apiary="description" />
+
 </section>
 
 <h2>Who we are</h2>
 
-<p>The Verifiable Claims Working Group (VCWG) co-chairs are Matt Stone, Pearson; Richard Varn, ETS, and Daniel Burnett, Invited Expert.</p>
+<p>The Verifiable Claims Working Group (VCWG) co-chairs are:</p>
+
+<div data-apiary="chairs ${name}"></div>
 
 The W3C team contact is <a href="mailto:liam@w3.org">Liam Quin</a>.</p>
```

Comment:
@tripu can't you also retrieve the Team contacts info from the W3C api ?

Reply:
@vivienlacourba: yes, but Apiary shows only simple, scalar values as returned by the API (in this case, [any of these fields](https://api.w3.org/groups/98922/teamcontacts?embed=true)).

On the original page, Liam's name is a hyperlink to his e-mail address; I can't replicate that.

I can get the list of team contacts from with Apiary, and show that&nbsp;&mdash;&nbsp;but there won't be links to their e-mail addresses.

Reply:
@tripu true, for privacy reason we decided not to expose in the W3C API sensitive information live email addresses. So you can forget my comment.

Reply:
Thanks for reviewing, @vivienlacourba :)